org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/views/issues/OnTheFlyIssuesView.java

```diff
@@ -26,12 +26,12 @@
 import org.eclipse.swt.widgets.Label;
 import org.sonarlint.eclipse.ui.internal.SonarLintUiPlugin;
 
-public class IssuesView extends MarkerViewWithBottomPanel {
+public class OnTheFlyIssuesView extends MarkerViewWithBottomPanel {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 7 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)